packages/shared/src/libs/enums/exception-message.enum.ts

```diff
@@ -1,5 +1,9 @@
 const ExceptionMessage = {
-	CREATE_PROJECT_GROUP_FAILED: "Failed to create project group",
+	CONTRIBUTOR_NOT_FOUND: "Contributor not found.",
+	CREATE_ACTIVITY_LOG_FAILED: "Failed to save activity logs.",
+	CREATE_PROJECT_GROUP_FAILED: "Failed to create project group.",
+	EMAIL_USED: "Email address is already in use.",
+	GIT_EMAIL_NOT_FOUND: "Git email not found.",
```

Comment:
We decided to stick to the format: `SOMETHING_ACTION_RESULT` -- for example `USER_CREATE_SUCCESS`, so all the messages will be "grouped" (place near each other) by the object, they relate to
```suggestion
	CONTRIBUTOR_NOT_FOUND: "Contributor not found.",
	ACTIVITY_LOG_CREATE_FAILED: "Failed to save activity logs.",
	PROJECT_GROUP_CREATE_FAILED: "Failed to create project group.",
	EMAIL_USED: "Email address is already in use.",
	GIT_EMAIL_NOT_FOUND: "Git email not found.",
```